Dockerfile

```diff
@@ -112,7 +112,7 @@ RUN mkdir -p /var/run/tailscale /var/cache/tailscale /var/lib/tailscale
 COPY limits.sh /app/limits.sh
 ENTRYPOINT ["/app/limits.sh"]
 
-CMD ["/tailscale/wrapper.sh"]
+CMD ["sh", "-c", "/tailscale/wrapper.sh && /app/bin/migrate && /app/bin/server"]
```

Comment:
We should check the Fly's behavior in the case when migration unsuccessful

Reply:
@abc3 looks like Fly's release command "perform database migrations" and "run in a temporary VM that is created just before your application is deployed and released" so it helps with "zero-downtime deploys" but what about local development or self-hosting that's not on Fly? How will we run db migrations then?

Reply:
@abc3 I added back the release command and will take care of migrations `docker-compose.yml`.